rpc/gnmi/mixed-schema.md

```diff
@@ -138,7 +156,7 @@ replace: <
     `
   >
 >
-replace: <
+update: <
```

Comment:
What happens if I make a set request with two `update`s of origin `cli`?

Reply:
This is currently undefined.  To obtain defined behavior, I suggest combining the single setRequest with 2 or more CLI updates into a single setRequest with a single CLI update.